### ISSUE TITLE
2024_10 BUG: when a team member logs into the Sim UI (PROD), the world shown is incorrect and the team name is not displayed 

### ISSUE THREAD

**Issue by ljmoody** [lzim/teampsd#3253]
Discovered 9/5/24 while updating manual instructions, screenshots, etc.

- When logging on as a team member, no matter which world I choose, the Sim UI is displaying "Shared" in the Team Name field. 
- There is also no indication of which team I am in, which could be confusing for users on multiple teams. _How will they know what team they are currently logged into?_
- **This does not apply to administrator logins/accounts.**

**Current behavior:**
No matter which team or world I log into, the Team Name field displays the same thing:
![image](https://github.com/user-attachments/assets/85fa2b75-ea4a-436c-8eab-141b546af0e8)

**Desired behavior:**
- [x] At a minimum, the world displayed in the Team Name field should be the world in which the team member selected while logging in. 
- [x] _Would it also be helpful to display the full or partial team name, at least, to remind/orient the team member to which team (and world) they are logged into?_ 
---
- [x] Current documentation in _MTL_ Manual - may need to change based on effort associated with this card.
![image](https://github.com/user-attachments/assets/c630ae9e-64c0-44df-8d0e-66e1d92898dc)

**Comment by matthewtomo** [comment]
**Discussed Dev Standup wk_1 2024_9**
@matthewtomo @tejaspesquare 

- Dev has been able to reproduce this behavior.
- They are changing the text to show the world you selected at login. 
- Once testing is complete, they will be able to deploy those changes to PROD.

**Comment by tejaspesquare** [comment]
@ljmoody ,
This has been fixed & available on PROD. Kindly note that this change is not available on TEST as couple of development is in-progress on local environment.

- _Team Login



- Admin Dashboard
![image](https://github.com/user-attachments/assets/8bebc527-905c-4e59-b278-bc07d2e0c125)

**Comment by ljmoody** [comment]
I believe this card was closed prematurely/accidentally. Testing today in Chrome, TEST environment, on VA laptop.

##  Logged in as administrator: ✅ 
![image](https://github.com/user-attachments/assets/6487e942-a237-4c9e-b999-dfc39c5c58ef)

## Logging in as member of jenns_2nd_test_world_03_10_24, _ind world: ✅ 
![image](https://github.com/user-attachments/assets/1e45e424-9cda-4d27-aee7-327a919701b8)
- [ ] **NOTE:** @ljmoody needs to revise the manual:
![image](https://github.com/user-attachments/assets/266c9f1b-9992-407d-b08a-deae4e4804ba)

## Logging in as member of jenns_2nd_test_world_03_10_24, _team world: ✅ 
![image](https://github.com/user-attachments/assets/ade8b529-cad7-44b5-9ec8-98fd8688d7c5)

## Logging in as member of 3220_test_edge, _shared world: ❓ 
@matthewtomo and @tejaspesquare, I believe users should also see the team name along with the _shared extension here - correct?
![image](https://github.com/user-attachments/assets/b4d8ff20-382c-44f4-a110-504298b320fb)

**Comment by tejaspesquare** [comment]
Hello @ljmoody ,
> I believe this card was closed prematurely/accidentally. Testing today in Chrome, TEST environment, on VA laptop.

> ## Logged in as administrator: ✅
> ## Logging in as member of jenns_2nd_test_world_03_10_24, _ind world: ✅
> ## Logging in as member of jenns_2nd_test_world_03_10_24, _team world: ✅
> ## Logging in as member of 3220_test_edge, _shared world: ❓

This matter has been addressed and is currently accessible in the Development environment; however, it has not yet been transferred to the TEST instance.

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk_1 2024_10**
@matthewtomo @tejaspesquare 

- This fix is currently only available in DEV
- @ljmoody we will refrain from testing it there until it can be promoted to TEST

**Comment by lzim** [comment]
@matthewtomo 

I was trying to find the PREPPED Blue Teams in Sim UI PROD today and I could not find a workable team from the past or present.
I adjusted the Title to ANDON since it's creating a work stoppage and we need to send the #consult_workflow invites out.
Put in the top of the agenda for #9am_workflow leads please. Thx!

**Comment by tejaspesquare** [comment]
@ljmoody , @matthewtomo 
> * This fix is currently only available in DEV
> * @ljmoody we will refrain from testing it there until it can be promoted to TEST

Code has been promoted & can be verified on TEST environment.

<ins>_shared world</ins>

![image](https://github.com/user-attachments/assets/c2d40a8a-67c8-4b27-bc37-1950e0050b61)

<ins>_ind world</ins>
![image](https://github.com/user-attachments/assets/0583d422-4472-4b50-b131-a6e0f2943625)

<ins>_team world</ins>
![image](https://github.com/user-attachments/assets/cb3d4f92-0bab-418d-bee9-92d2f8cc0a0e)

**Comment by lzim** [comment]
@matthewtomo 
@ljmoody 

Please add these to our agenda to review on https://mtl.how/2025_roadmap at **#9am_workflow leads**

**I can only join first 30 minutes of #sim_ui_workflow this week.**

**Comment by matthewtomo** [comment]
> @matthewtomo
> 
> I was trying to find the PREPPED Blue Teams in Sim UI PROD today and I could not find a workable team from the past or present. I adjusted the Title to ANDON since it's creating a work stoppage and we need to send the #consult_workflow invites out. Put in the top of the agenda for #9am_workflow leads please. Thx!

Hi @lzim, 

I don't believe that the issue you are describing in this comment is related to the main issue of this card.

To clarify, are there teams that have been prepped which you are unable to locate in the Sim UI?

**Comment by lzim** [comment]
No @matthewtomo - I was trying to track what this card was about. Please edit to reflect your understanding. I'll be able to see the correction and understand.
Looks like @ljmoody is the card author if she can help too.
Thx!

**Comment by ljmoody** [comment]
@lzim and @matthewtomo - taking a few educated guesses here, but pretty sure this message Lindsey posted over the weekend is what Matthew & I were trying to understand - and thought, as written, was related to another card or was potentially another bug altogether: 
> I was trying to find the PREPPED Blue Teams in Sim UI PROD today and I could not find a workable team from the past or present. I adjusted the Title to ANDON since it's creating a work stoppage and we need to send the #consult_workflow invites out. Put in the top of the agenda for #9am_workflow leads please. Thx!

We didn't understand how this card was promoted to an ANDON or how it was creating work stoppage.

_Does the above message mean you (@lzim) couldn't find those teams in the Sim UI? Or do you mean that that sites you prepped are also having the behavior related to this card (the team name not displaying correctly? Or do you mean something else entirely?_

The behavior that made me create this card was:  the world displayed in the Team Name field should be the world in which the team member selected while logging in. 

The latest update from Dev looks like this has been fixed and is in TEST instance. Testing will be the next step, then if verified, it'll migrate to PROD.

**Comment by ljmoody** [comment]
**10/9/2024** - tested in TEST instance (**Chrome & Edge** browsers, VA laptop) to verify behavior before revising information about Team Name element/field in _MTL_ Manual.
1. Logged in as Administrator: ✅ 
![image](https://github.com/user-attachments/assets/33b99849-da13-4bac-b13d-df4e1089908c)

2. Logged in as member of jenns_2nd_test_world_03_10_24, _ind world: ✅ 
![image](https://github.com/user-attachments/assets/d3cec9d6-00cd-41ce-8221-8ace29da09a4)

3. Logged in as member of jenns_2nd_test_world_03_10_24_team, _team world: ✅ 
![image](https://github.com/user-attachments/assets/605ce00a-52b0-4aee-b2a4-3086841c59a3)

4. Logged in as member of june12_laura_testing, _shared world: ✅ 
![image](https://github.com/user-attachments/assets/03f7d3f8-7a4f-4363-935d-e31bf7e81c36)

- Adjusted points to reflect effort for this task.
- Made update to _MTL_ Manual.
- Closing this card; work is compete!


fyi @matthewtomo @tejaspesquare @lzim @jamesmrollins

**Comment by ljmoody** [comment]
Tested today in PROD using both Edge and Chrome - same results.
✅ verified team name and world are correctly displayed upon logging into the Sim UI

![image](https://github.com/user-attachments/assets/bb3284c8-5111-4fc9-a718-fa96d6fd85d8)

fyi @matthewtomo @lzim
